.circleci/config.yml

```diff
@@ -94,8 +94,11 @@ jobs:
             set -x
             mkdir -p junit
             go test -v ./... 2>&1 | tee tests.log
-            go-junit-report < tests.log > junit/influxdb.junit.xml
           no_output_timeout: 1500s
+      - run:
+          name: Convert test output to junit
+          command: go-junit-report < tests.log > junit/influxdb.junit.xml
```

Comment:
Fix to always report which tests actually failed (especially when a test actually does fail).